aries-site/src/examples/cardPreviews/layer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Box } from 'grommet';
+
+export const LayerPreview = () => {
+  return (
+     <Box fill direction='row'>
+         <Box basis='3/4' background="brand"/>
```

Comment:
This feels a bit loud in comparison to the rest. What if we used `background-contrast` or something instead?